codegen/tests/simple/server.go

```diff
@@ -0,0 +1,91 @@
+package simple
+
+import (
+	"context"
+
+	"github.com/anz-bank/pkg/log"
+	"github.com/anz-bank/sysl-go/config"
+	"github.com/anz-bank/sysl-go/core"
+	"github.com/anz-bank/sysl-go/handlerinitialiser"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// GENERATED: SERVER
+
+type Server struct {
+	params           *core.ServerParams
+	serviceInterface *ServiceInterface
+	restGenCallback  core.RestGenCallback
+	grpcGenCallback  core.GrpcGenCallback
+}
+
+func StartServer(file string, customConfig interface{}, build func(*config.DefaultConfig) (*Server, error)) error {
+	cfg := NewDefaultConfig()
+	err := config.LoadConfig(file, &cfg, &customConfig)
```

Comment:
```suggestion
	err := config.LoadConfig(file, &cfg, customConfig)
```
IMO the passed in customConfig supposed to be the address of the custom config variable.

Reply:
Thanks. I suspect you're right. The proposal is just structural. I haven't actually executed it :)